### ISSUE TITLE
Old movies in Plex library shows as requested in Ombi 

### ISSUE THREAD

**Issue by henrikrox** [tidusjar/Ombi#1165]
#### Ombi Version:

	2.1.1336

#### Update Branch:

Stable

#### Media Sever:

Plex

#### Operating System:

Windows 10

#### Problem Description:

So after rolling back to stable, deleting ombi directory on c:/

I have noticed alot of movies, that are showing up as requested instead of available. Also movies that i have never requested (but are in my plex library)

For example

A beautifuld Mind is in Radarr (not being monitored) and on Plex.

However in Ombi it shows as requested, but its not in the request queue. Same with A monster Calls, and alot of other movies. Important to note i never requested those movies inside, they were already on/downloaded on plex before i tried out Ombi, but they are stilling showing up as requested.

Where does it get this info from? How can i start from scratch so that it syncs everything from Plex correctly. I cant manual  make them available either, since they arent in the request queue.

See pictures

#### Reproduction Steps:

![2](https://cloud.githubusercontent.com/assets/6884716/23158318/15de8ea0-f81f-11e6-8e55-411cdefe6b18.PNG)
![1](https://cloud.githubusercontent.com/assets/6884716/23158320/15e29ac2-f81f-11e6-8a4c-015b157ec04b.PNG)
![3](https://cloud.githubusercontent.com/assets/6884716/23158319/15e206ac-f81f-11e6-80d4-eee0a589aead.PNG)

**Comment by henrikrox** [comment]
Here can you see its not in any queue 

![das](https://cloud.githubusercontent.com/assets/6884716/23158409/63c44290-f81f-11e6-922c-2185ede72bae.PNG)

**Comment by SuperPotatoMen** [comment]
It is called "requested" beceause the system didn't sync with your plex library yet.
Can you set your log level to debug and run the plex cacher and checker manually?

**Comment by henrikrox** [comment]
I managed to get it manually sinc. So the requested went away in Ombi for the movies in question. But it wasnt marked as available. Figured it out why though.

Out of 500 plex movies i have, 30-40 were mistmatched in Ombi and Plex because of different release year. Tryin to add gave and error, to contact admin (because it was already in Radarr). If i manually deleted it from Radarr (but not the files), then requested it again from Ombi, then went to Radarr and pointed the movie to its already downloaded movie file. Ombi then saw the Movie as available.

I Did this manually for all the movies in question.

Dunno how Plex get their release dates/years from. Some old titles had 3-4 years difference in Plex vs Imdb/themoviedb.  Imdb and themoviedb usually matched, but Plex was different even though its pulling from those sources.

Maybe its because of the local media assets agent in Plex? Dunno.

At least it works now :)
